### PR TITLE
Update TF whisper doc tests

### PR DESCRIPTION
**Pull request by amyeroberts**
# What does this PR do?

Fixes doctests for the TF whisper model. 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: src/transformers/models/whisper/modeling_tf_whisper.py [hunk 1]
```diff
@@ -1288,7 +1284,7 @@ def call(
         >>> inputs = processor(ds[0]["audio"]["array"], return_tensors="tf")
         >>> input_features = inputs.input_features
 
-        >>> generated_ids = model.generate(inputs=input_features)
+        >>> generated_ids = model.generate(input_ids=input_features)
```
Comment (ydshieh) [inline src/transformers/models/whisper/modeling_tf_whisper.py:1287]: Is this change necessary? I believe it works with `inputs` (and should work if not).

Comment (amyeroberts) [reply src/transformers/models/whisper/modeling_tf_whisper.py:1287]: I believe so. I'm unable to pass in `inputs` to the tensorflow model generate function and this matches with what I see in the code. 

[PT generate function](https://github.com/huggingface/transformers/blob/df8faba4db89897e6a1da4b28a17c01aa24e3b7e/src/transformers/generation_utils.py#L917) takes `inputs`

[TF generate function](https://github.com/huggingface/transformers/blob/df8faba4db89897e6a1da4b28a17c01aa24e3b7e/src/transformers/generation_tf_utils.py#L380) takes `input_ids`. `inputs` would be passed in as `model_kwargs`

I agree that the inputs should be standardised if they're not already. I think that might be a bit of work to make sure it's updated safely if it needs to be cc @gante

**Review by ydshieh — APPROVED** [review]
Thank you again, @amyeroberts !

File: src/transformers/models/whisper/modeling_tf_whisper.py (hunk 1 above)
Comment (gante) [reply src/transformers/models/whisper/modeling_tf_whisper.py:1287]: (added to the TODO list -- I was aware of the problem but hadn't placed it there :) )

**Review by sgugger — APPROVED** [review]
Thanks for fixing!
